modules/nux/modules.php

```diff
@@ -71,12 +72,20 @@ public function process() {
         if (data_source_available($modules, 'smtp')) {
             $smtp_servers = count(Hm_SMTP_List::dump(false));
         }
```

Comment:
Looks like the smtp section got duplicated here and we should set a default empty value for $profiles if the site does not have that module set enabled.